### ISSUE TITLE
Incorect usage of reduce method

### ISSUE THREAD

**Issue by sergiuchilat** [xvolik/PowerIT#7]
https://github.com/xvolik/PowerIT/blob/34c91f460bf7c6ab80948a9e6953eaa78b2f8835/Arrays/task%201.html#L60

**Comment by xvolik** [comment]
Yeah, .reduce has other function but here is used as another way and it works :)
"Найдите еще один способ вывода массива"
